autobatching_max_keys typed as bool in create_client

As the title says, in function `create_client` the parameter `autobatching_max_keys` is wrongly typed as `bool` instead of `int` (client.py:635). 

So mypy shows an error, e.g.
`error: Argument "autobatching_max_keys" to "create_client" has incompatible type "int"; expected "bool"  [arg-type]`

--- a/emcache/client.py
+++ b/emcache/client.py
@@ -632,7 +632,7 @@ async def create_client(
     cluster_events: Optional[ClusterEvents] = None,
     purge_unhealthy_nodes: bool = DEFAULT_PURGE_UNHEALTHY_NODES,
     autobatching: bool = DEFAULT_AUTOBATCHING_ENABLED,
-    autobatching_max_keys: bool = DEFAULT_AUTOBATCHING_MAX_KEYS,
+    autobatching_max_keys: int = DEFAULT_AUTOBATCHING_MAX_KEYS,
     ssl: bool = DEFAULT_SSL,
     ssl_verify: bool = DEFAULT_SSL_VERIFY,
     ssl_extra_ca: Optional[str] = None,

